assets/less/hel/hearing-page.less

```diff
@@ -4,6 +4,26 @@
     font-size: 1.4rem;
     margin-bottom: 25px;
 
+    .language-link {
+        padding: 4px;
+        border-radius: 4px;
+
+        a {
+            cursor: pointer;
+        }
+
+        &-active {
+            padding: 4px;
+            border-radius: 4px;
+            background: #04a1d4;
```

Comment:
Perhaps use dark gray background and white text for the active element. I.e. the same colors as in all the labels in the hearings. I think it fits the sidebar better.